### ISSUE TITLE
MDBOOK007: false error for an anchor name of 3 characters or less

### ISSUE THREAD

**Issue by n8behavior** [joshrotenberg/mdbook-lint#498]
MDBOOK007 reports an error for a valid `{{#include}}`. mdBook renders the same
include correctly.

**inc.sh**

```bash
# ANCHOR: abc
echo x
# ANCHOR_END: abc
```

**src/p.md**

```markdown
{{#include ../inc.sh:abc}}
```

**mdbook-lint lint --enable MDBOOK007 src/p.md**

```text
error[MDBOOK007]: Invalid line number format 'abc'. Expected number or number:number format.
```

The length of the anchor name decides it. `a`, `ab`, and `abc` give the error.
`abcd` and longer names do not.

mdbook-lint 0.16.1, mdbook 0.5.4.
